packages/core/src/checkbox/checkbox.element.ts

```diff
@@ -39,8 +39,8 @@ export class CdsCheckbox extends CdsInternalControlInline {
     return [...super.styles, styles];
   }
 
-  connectedCallback() {
-    super.connectedCallback();
+  firstUpdated(props: Map<string, any>) {
+    super.firstUpdated(props);
```

Comment:
Most of these changes in this commit are for frameworks like Angular that may delay rendering slotted elements.
Sometimes Angular will render the host component first then render the slotted items at a later point. Some of the code in the forms was querying in `connectedCallback` when the component is created but the slotted items have not yet been created.

The fix is to push the logic into a later lifecycle hook `firstUpdated` which is called after the component has fully rendered and updated one time. At this point angular has rendered the entire tree and the web component has full rendered so it is safe to start querying the slots for the native inputs.

Reply:
Curious if we run into similar problems for Vue and React (my guess is yes) and if so, have you verified that `firstUpdated` is sufficiently late enough across different frameworks?

Reply:
Yeah the same fixed works for the other frameworks from what I have tested so far. I added a simple input example to the vue and parcel demos but will wait on React so theres no merge conflicts with the react wrappers pr